mitodl/open-discussions
# Added mappings for some new MITPE topics

@pdpinch the mappings we discussed

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #3178 

#### What's this PR do?
Maps a couple new topics that showed up

#### How should this be manually tested?
Run `./manage.py backpopulate_mitpe_data` and verify courses that have these topics on https://professional.mit.edu/course-catalog are correctly mapped.
